setup.sh

```diff
@@ -252,7 +252,8 @@ function install() {
     echo "-------------------------"
     echo "Run the bot !"
     echo "-------------------------"
-    echo "You can now use the bot by executing 'source .env/bin/activate; freqtrade trade'."
+    echo "You can now use the bot by executing 'source .env/bin/activate; freqtrade <command>'."
+    echo "You verify that freqtrade is installed successfully by running 'source .env/bin/activate; freqtrade --version'."
 }
```

Comment:
* Maybe add here a third line:
```
echo "You can now use the bot by executing 'source .env/bin/activate; freqtrade <command>'."
echo "You can see the list of available bot commands by executing 'source .env/bin/activate; freqtrade --help'."
echo "You verify that freqtrade is installed successfully by running 'source .env/bin/activate; freqtrade --version'."
```
?

* "command" --> "subcommand"?
(just to be aligned to the term we took from argparse and use in other places in the docs)